src/ProducerSideService/include/ProducerSideService/ProducerSideServer.h

```diff
@@ -1,9 +1,9 @@
-// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef ORBIT_SERVICE_TARGET_SIDE_SERVER_H_
-#define ORBIT_SERVICE_TARGET_SIDE_SERVER_H_
+#ifndef ORBIT_PRODUCER_SIDE_SERVICE_TARGET_SIDE_SERVER_H_
+#define ORBIT_PRODUCER_SIDE_SERVICE_TARGET_SIDE_SERVER_H_
```

Comment:
Ehm, I guess this shows how I had originally called the class... Could you adjust to `ORBIT_PRODUCER_SIDE_SERVICE_PRODUCER_SIDE_SERVER_H_` (also the comment at the end of the file)?

Reply:
Fixed accordingly! Thanks!